ga4gh/frontend.py

```diff
@@ -384,8 +403,139 @@ def searchDatasets(version):
         version, flask.request, app.backend.searchDatasets)
 
 
+# The below paths have not yet been implemented
+
+
+@app.route('/<version>/callsets/<no(search):id>')
```

Comment:
Ugh, this is pretty horrible. So, we can never have an object with ID = 'search'? Is this a standard RESTish issue do you think, or something new we've made up ourselves?

There's not a lot we can do about it though. Can you add a quick comment on the Converter class to explain why we need this? I'm happy to merge then.